appdatabase/migrations/sql/1674570455_communities_permissions.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS community_permissions (
+    community_id TEXT NOT NULL,
+    permission_id VARCHAR NOT NULL PRIMARY KEY,
+    permission_private BOOLEAN NOT NULL DEFAULT FALSE,
+    is_allowed_to INT NOT NULL DEFAULT 0
```

Comment:
You're missing the condition in this. For example, we have `is_allowed_to` that can be `admin`, but we don't have how people can become that.

There is "holds token", "holds collectible", maybe others. We need to add this.

I think starting with just holding >1 token of something is the simplest starting point, but it can be a collectible since that's what Michal started on for minting

Reply:
We have 3 types of "hold": Assets, Collectibles and Ens. So, in case of Assets and Collectibles permission will have "amount" field, and in case of Ens, there will be string.